Update basic-package-mgmt.md (#7084)
NixOS/nix: doc/manual/src/package-management/basic-package-mgmt.md

> The flag `-q` specifies a query operation, `-a` means that you want
> to show the “available” (i.e., installable) packages, as opposed to the

Reviewer: ```suggestion
The flag `-q` specifies a query operation, `-a` shows packages 
“available” to install instead of those already
```

> installed packages, and `-P` prints the attribute paths, that can be used

Reviewer: ```suggestion
installed, and `-P` prints attribute paths that you can use
```

> to unambiguously select a package for installation. In the example above,

Reviewer: ```suggestion
to unambiguously select a package to install. In the attribute path
```

> `nixpkgs`is the channel, and `aterm` is the

Reviewer: ```suggestion
`nixpkgs.aterm`, `nixpkgs` is the channel and `aterm` is the
```

> [derivation](../language/derivation.md) name in the attribute path,

Reviewer: ```suggestion
[derivation](../language/derivation.md) name.
```

> `nixpkgs.aterm`. If you downloaded Nixpkgs yourself, or if you checked it

Reviewer: ```suggestion
If you downloaded Nixpkgs or checked it
```
